airflow/dags/gtfs_views/gtfs_schedule_history_calendar_long.yml

```diff
@@ -0,0 +1,26 @@
+operator: operators.SqlToWarehouseOperator
+dst_table_name: "views.gtfs_schedule_history_calendar_long"
+dependencies:
+  - warehouse_loaded
+
+sql: |
```

Comment:
how does this templating get processed? Jinja?

Reply:
yeah--airflow operators have a `template_fields` argument, that allows the listed arguments to be jinja template strings! It's pretty similar to how https://www.getdbt.com/ works!